src/types/DBEventType.ts

```diff
@@ -0,0 +1,7 @@
+export type DBEvent = {
+  id: string;
+  name: string;
+  createdAt: Date;
```

Comment:
We set some `createdAt` and `updatedAt` dates as strings, shall we keep them here as Dates?

Reply:
moved it to string